magento-research/pwa-studio
# bug fixes to make aws pr-test build work same as cicrcle ci

How to test - Fail conditions - 
1. To test yarn install --frozen-lockfile || exit 1 
   Change "babel-eslint": "~10.0.1" to    "babel-eslint": "~15.0.1"   and git Push then observe aws log.

2. To test yarn run build || exit 1
   packages/venia-concept/src/components/SearchBar/__stories__/searchBar.js  make some compilation errors like remove (    and git Push then observe aws log

3. To test yarn workspace @magento/venia-concept run storybook:build || exit 1
 packages/venia-concept/src/components/SearchBar/__stories__/searchBar.js   change import { SearchBar } from '../searchBar';  to import { SearchBar } from '../searchBar/adjshsf';    and git Push then observe aws log

4. To test yarn run danger    packages/venia-concept/src/__tests__/example.spec.js  change     expect(true).toBe(true); to     expect(true).toBe(false);

Pass Condition - 
If there are no errors then "yarn run danger "  should exit with status code 0
